JustBurp/myapp/lib/loadChefDailyMenu.js

```diff
@@ -1,16 +1,15 @@
 var DailyMenu = require('../models/dailyMenu');
```

Comment:
From my understanding, pushing three dailyMenu to the DailyMeunsContext will make default inputs for the user in all three days. I am wondering what would happen in the **chefDailySetupCard** if one chef does not have the currentDailyMenu and chef.dailyMenus.length == 0?  Will we render any datesContext to the frontend in this case?  Please correct me if I understand wrong.